src/gui/editors/PianoRoll.cpp

```diff
@@ -3918,27 +3918,32 @@ int PianoRoll::quantization() const
 
 void PianoRoll::quantizeNotes()
 {
+	if( ! hasValidPattern() )
+	{
+		return;
+	}
+
 	NoteVector notes = getSelectedNotes();
 
-	if (notes.empty())
+	if( notes.empty() )
```

Comment:
This is a new function, and therefore the old coding style (wrt spaces inside parentheses) should not apply. I realize this is late, but still 😅